### PR TITLE
sundararajan's update 2

### PR DESCRIPTION
**Pull request by sundararajan20**
* Final validation loss: 3.59
* Link: https://api.wandb.ai/links/sundararajan-team/2w0t40kv
* Description: torch.compile + vanilla model with 8 layers + weight tying of input and output embeddings + hyperparameter optimization

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
